src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicWithHorizon.hpp

```diff
@@ -90,7 +94,8 @@ struct EvolutionMetavars<3, InitialData, BoundaryConditions>
                    StrahlkorperTags::UnitNormalVectorCompute<frame>,
                    StrahlkorperTags::GradUnitNormalOneFormCompute<frame>,
                    StrahlkorperTags::ExtrinsicCurvatureCompute<frame>,
-                   StrahlkorperGr::Tags::SpinFunctionCompute<frame>>,
+                   StrahlkorperGr::Tags::SpinFunctionCompute<frame>,
+                   StrahlkorperTags::RicciScalarCompute<frame>>,
```

Comment:
This Tag should be one that computes the Ricci scalar from the Ricci tensor by contracting with the metric (the Ricci tensor is already available, so why recompute it), but I think RicciScalarCompute uses derivs of Christoffel.